<launch>

 <!-- GAZEBO-->
      <arg name="paused" default="false"/>
      <arg name="use_sim_time" default="true"/>
      <arg name="gui" default="true"/>
      <arg name="headless" default="false"/>
      <arg name="debug" default="false"/>

   <include file="$(find gazebo_ros)/launch/empty_world.launch">
	    <arg name="world_name" value="$(find summit_xl_gazebo)/worlds/summit_xl.world"/>
	    <arg name="paused" value="$(arg paused)"/>
	    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
	    <arg name="gui" value="$(arg gui)"/>
	    <arg name="headless" value="$(arg headless)"/>
	    <arg name="debug" value="$(arg debug)"/>
  </include>
 <!--ROBOT1-->
 <group ns="robot1">
  	
  	 <node pkg="ed_pmov" type="pmov" name="pmov" output="screen" /
	 <include file="$(find dynamixel_moveit_arm)/launch/demo.launch" >
		 <arg name="z" default="0.05"/>
	 	 <arg name="x" default="0.0"/> <!-- -1.5-->
	 	 <arg name="y" default="0.0"/>
 	 </include>
 </group>
 	  
 	   <!-- ROBOT2-->

  <!--	<include file="$(find arm_dual_moveit)/launch/camara.launch"/>
<include file="$(find arm_dual_moveit)/launch/camara.launch"/>-->


</launch>
